src/plugins/chart_expressions/expression_metric/common/types/expression_renderers.ts

```diff
@@ -28,7 +28,7 @@ export interface MetricVisParam {
   percentageFormatPattern?: string;
   metricColorMode: ColorMode;
   palette?: CustomPaletteState;
-  labels: Labels;
+  labels: Labels & { style: Style; position: 'bottom' | 'top' };
```

Comment:
Could you, please, move it to the separate type `LabelsConfig`?

Reply:
done